File: Schedule.Core/DTO/Account/UserDto.cs
using Schedule.Core.Enums;
using System;

namespace Schedule.Core.DTO.Account
{
    public class UserDto
    {
        public Guid Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string GroupName { get; set; }
        public Role Role { get; set; }
    }
}

File: Schedule/ClientApp/App/src/app/studying/models/UserDto.ts
import { Guid } from 'guid-typescript/dist/guid';
import { Role } from 'src/app/core/enums/role.enum';

export class UserDto {
  id: Guid;
  email: string;
  firstName: string;
  lastName: string;
  phone: string;
  role: Role;

  constructor(model: Partial<UserDto>) {
    this.id = model.id;
    this.email = model.email;
    this.firstName = model.firstName;
    this.lastName = model.lastName;
    this.phone = model.phone;
    this.role = model.role;
  }
}
